scripts/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: '3.7'
+services:
+  artifactory-1:
+    image: releases-docker.jfrog.io/jfrog/artifactory-pro:7.27.10
```

Comment:
Should we use the latest instead of the hardcoded version?

Reply:
@danielmkn Are we using `latest` in CI?

Reply:
yes

Reply:
and we printing the version to the output